modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunQueryRoute.java

```diff
@@ -215,7 +215,7 @@ private List<RasRunResult> getRunsByIds(List<String> runIds, boolean isMethodDet
         return runs;
     }
 
-	private List<IRasSearchCriteria> getCriteria(RasQueryParameters queryParams) throws InternalServletException {
+	private List<IRasSearchCriteria> getCriteria(RasQueryParameters queryParams) throws InternalServletException, ResultArchiveStoreException {
```

Comment:
Is this new ResultArchiveStoreException caught somewhere? Might be worth having a try-catch to only have an InternalServletException being thrown